packages/block-editor/src/components/block-list/use-block-props/use-event-handlers.js

```diff
@@ -35,7 +35,7 @@ export function useEventHandlers( clientId ) {
 			} = select( blockEditorStore );
 
 			return {
-				isSelected: isBlockSelected( clientId ),
```

Comment:
I've seen this pattern in a few places and it kind of makes sense: only call the selector when it's needed. I remember discussing this with @youknowriad and Andrew in the context of performance. I wonder, is there any way we can make this more "official" as in make it look ok to do, not odd/normal looking, and maybe even recommend doing it? This is usually data needed for event handling rather than data needed to render the element tree.